wiki/Beatmap_Discussion/en.md

```diff
@@ -126,38 +132,57 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
 ### Functions
 
 - `Collapse all`/`Expand all` and the up arrow: These will show/hide posts.
-- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on suggestions or problems.**
+- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on Suggestions or Problems.**
 - Timeline: Shows the timestamp the issue addresses. Only available in the Timeline section.
 - CREATOR icon: Displays responses/submissions from the user who made the beatmapset.
 - permalink, edit, delete: Permalink grabs a direct link to the post. Edit allows the submitter to edit their submission. Delete allows the submitter to remove their submission.
 - Respond: Reply to the submission. Pressing Enter or clicking Reply afterwards will submit your response.
 - Mark as Resolved: The creator may select this when the issue has been addressed and either accepted or denied.
 - Reopen a Discussion: If the submission needs further discussion, the creator or mod submitter may reopen the submission by responding and unchecking Resolved.
 
-## Earning Kudosu!
+## Hype, Nomination, and Kudosu!
+
+### Hype
 
-*Note: You get ten votes every hour.*
+Hype is used to help push beatmaps forward to nomination; similar to how Star Priority was previously used. Once a beatmapset receives 5 Hype, it becomes eligible for Nomination. Please note the following:
+
+- Each user has a limited supply of hype (5 per month to start).
+  - You will regain a hype 30 days after that one hype was used.
+- Hype can only be used after playing through a difficulty from a beatmapset.
+
+### Nomination
+
+Nomination is similar to the previous modding system. A beatmap must receive 2 nominations from two [Beatmap Nominators](/wiki/Beatmap_Nominators) before being automatically moved to the qualification stage. Beatmaps must have all Suggestions and Problems resolved before it can get ranked. BN/QAT now can reset nominations by posting a problem in the discussion field.
+
+### Kudosu!
 
 ![](img/thumbs.jpg "Thumbs up for Kudosu!")
 
-Modders can now earn Kudosu! by having a thumbs up on their mod. You can (and probably should) go through old mods on your beatmaps and give them a thumbs up to support the modders before this feature was implemented.
+Giving a thumbs up to a suggestion or problem in the discussion field will give the modder Kudosu!. The amount of thumbs up received determines how much Kudosu! the modder gets. Any user besides the poster can thumbs up any post. *Note: You get 60 votes every hour.*
```

Comment:
Currently worded like you should do 60 votes per hour, should say it's limited or capped to 60 votes per hour
Also, BN/QAT don't have this limit.

Reply:
whoops, missed this... is it just BN and QAT, not GMT?

Reply:
oh wait, never mind... will reword to `*Note: You are limited to 60 votes per hour, while BN and QAT members do not have this limit.*`